app-nordigen/README.md

```diff
@@ -0,0 +1,61 @@
+# Integration new bank
+
+Find in [doc](https://docs.google.com/spreadsheets/d/1ogpzydzotOltbssrc3IQ8rhBLlIZbQgm5QCiiNJrkyA/edit#gid=489769432) what is id of bank which you want to integrate
```

Comment:
Is this an official publication? (Or where could this spreadsheet be updated from in the future?)

Reply:
Yes, it's official. It's listed here in the official documentations for "outputs" - https://nordigen.com/en/docs/account-information/output/accounts/